packages/css-library/src/stylesheets/base/va.scss

```diff
@@ -21,11 +22,16 @@ body {
   // font-size: $base-font-size;
   font-size: 16px;
   font-family: inherit;
+  line-height: $vads-font-line-height-default;
```

Comment:
We need this because some elements have a line-height value of 1.5 which is set by a global foundation stylesheet. Our typographic elements specified in `uswds-typogrpahy.scss` get this by default, but something like a div doesn't. 

Here's an example of a div in profile that relies on default foundation global styles: 
![image](https://github.com/user-attachments/assets/3740f958-3c45-42a0-ab57-1f13a6f2b433)